### DISCUSSION TITLE
WSA上激活失败

### DISCUSSION THREAD

**Discussion by HumanMus1c** [iamr0s/Dhizuku#20] [General]
报错
[adb shell dpm set-device-owner.txt](https://github.com/iamr0s/Dhizuku/files/12098715/adb.shell.dpm.set-device-owner.txt)
![Snipaste_2023-07-20_01-49-17](https://github.com/iamr0s/Dhizuku/assets/46023154/25922362-c87d-435f-a251-e1f0efce52f3)

**Comment by iamr0s** [comment]
WSA阉割了设备所有者。这种问题常见于手表、电视机系统。

**Reply by HumanMus1c** [reply]
这太糟糕了
